CMakeLists.txt

```diff
@@ -2,13 +2,21 @@ project(ffxiv-wireshark-dissector C)
 
 cmake_minimum_required(VERSION 3.7)
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
-set(CMAKE_INSTALL_LIBDIR ~/.wireshark)
+IF (WIN32)
+  set(CMAKE_INSTALL_LIBDIR $ENV{APPDATA}/Wireshark/plugins)
+ELSE()
+  set(CMAKE_INSTALL_LIBDIR ~/.wireshark)
+ENDIF()
 
 include(UseMakeDissectorReg)
 include(CMakeForceCompiler)
 
 set(GLIB2_MIN_VERSION 2.4.0)
 
+IF(EXTRA_LIB_DIRS)
+  LINK_DIRECTORIES(${EXTRA_LIB_DIRS})
```

Comment:
What does this actually get used for? See above comment on modifying this file at all.